frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "moment-timezone": "0.5.27",
     "vue": "2.6.10",
     "vue-class-component": "7.1.0",
+    "vue-page-title": "^1.1.5",
```

Comment:
Pin the version. I believe it is possible to configure `npm` to always add a fixed version instead of a range during install. Don't forget to update the lock file.